control/src/select25.scss

```diff
@@ -293,6 +293,10 @@ $dropdown-padding: 8px 10px;
         width: 30px;
     }
 
+    &.s25-offscreen {
+        display: none;
```

Comment:
I dont think this is correct, it overloads the meaning of offscreen to mean hidden. I think the code should add the correct s25-hidden class when the control needs to be hidden.